### DISCUSSION TITLE
Disable formatting or forcing code block.

### DISCUSSION THREAD

**Discussion by cctl01** [oobabooga/text-generation-webui#2851] [Q&A]
I'm trying to have my virtual personal cycling coach prepare workouts for me. In order to be able to use them directly I need to have them in a certain format. Starting with a minus (-) sign.
By default the chat replies come formatted where an item in a list starting with a minus (-) sign gets replaced by a •. 

My current "workaround" is to ask for the training to be written in a 
`code block`
which sometimes works, but sometimes gets ignored. Would you know of a way to either;
**TL;DR in chat output how to:**
disable turning a - in to a •
always have the workout presented in a `code block`

**Comment by justoboy** [comment]
In the _start reply with_ textbox enter _```_.
